tests/test_doctestplus.py

```diff
@@ -351,7 +351,7 @@ def test_requires(testdir):
             >>> import foobar
         """
     )
-    testdir.inline_run(p, '--doctest-plus', '--doctest-rst').assertoutcome(passed=1)
+    testdir.inline_run(p, '--doctest-plus', '--doctest-rst').assertoutcome(skipped=1)
```

Comment:
This is testing that the test is skipped because of a missing import, I guess the outcome should be "skipped".

Reply:
how did all of these tests work before?

Reply:
See my comment below: https://github.com/astropy/pytest-doctestplus/pull/103#pullrequestreview-385227537